### PR TITLE
[Merged by Bors] - feat(AlgebraicGeometry): `Spec R[X] -> Spec R` is open

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/PrimeSpectrum/Polynomial.lean [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+Also see `AlgebraicGeometry/AffineSpace` for the affine space over arbitrary schemes.
+
+## Main results
+- `isNilpotent_tensor_residueField_iff`:
+  If `A` is a finite free `R`-algebra, then `f : A` is nilpotent on `κ(𝔭) ⊗ A` for some
+  prime `𝔭 ◃ R` if and only if every non-leading coefficient of `charpoly(f)` is in `𝔭`.
+- `Polynomial.exists_image_comap_of_monic`:
+  If `g : R[X]` is monic, the image of `Z(g) ∩ D(f) : Spec R[X]` in `Spec R` is compact open.
+- `Polynomial.isOpenMap_comap_C`: The structure map `Spec R[X] → Spec R` is an open map.
+- `MvPolynomial.isOpenMap_comap_C`: The structure map `Spec R[X] → Spec R` is an open map.
```
Comment (jcommelin) [inline Mathlib/AlgebraicGeometry/PrimeSpectrum/Polynomial.lean:26]: Maybe `R[X]` isn't the right notation in this setting.
I actually didn't know that this result was true without finiteness conditions on the number of variables. In particular, this doesn't follow from the standard lemmas (flat + fp => open, etc)

**Review by jcommelin — COMMENTED** [review]
bors d+

File: Mathlib/AlgebraicGeometry/PrimeSpectrum/Polynomial.lean (hunk 1 above)
Comment (erdOne) [reply Mathlib/AlgebraicGeometry/PrimeSpectrum/Polynomial.lean]: I changed it to `R[X̲]`. Hopefully it's better.

Yeah I didn't know this was true as well until Lean didn't ask for a finiteness assumption.

**Comment by erdOne** [comment]
Thanks for the review!
bors merge

**Pull request closed without merging**